### ISSUE TITLE
Template render error:  unknown block tag: elseif

### ISSUE THREAD

**Issue by flappyFeline** [hexojs/hexo#2515]
## Environment Info

Node version(`node -v`): `v7.7.1`

Your theme `_config.yml` (Optional): `Next`

Hexo and Plugin version(`npm ls --depth 0`): 
```
├── hexo@3.2.2
├── hexo-deployer-git@0.2.0 -> /Users/durand/Desktop/MyBlog/node_modules/.0.2.0@hexo-deployer-git
├── hexo-generator-archive@0.1.4
├── hexo-generator-category@0.1.3
├── hexo-generator-index@0.2.0
├── hexo-generator-tag@0.2.0
├── hexo-renderer-ejs@0.2.0
├── hexo-renderer-marked@0.2.11
├── hexo-renderer-stylus@0.3.1
└── hexo-server@0.2.0
```

## For BUG

 1.  BUG description: 
```
FATAL Something's wrong. Maybe you can find the solution here: http://hexo.io/docs/troubleshooting.html
Template render error: (unknown path) [Line 4, Column 152]
  unknown block tag: elseif
```
 2. The way to reproduce: 
After my test, if I type this code in my markdown file, it will be an error.
The code like these: 

```
` {% elseif theme.disqus_shortname %} `
```

Sorry about my bad English~

**Comment by flappyFeline** [comment]
If I use 
![image](https://cloud.githubusercontent.com/assets/16211378/25065134/05d26936-223c-11e7-9927-b582ecd75bbc.png)

to instead ``` ` {% elseif theme.disqus_shortname %} ` ```

Then do hexo c,hexo g, and it would be OK.

**Comment by NoahDragon** [comment]
It causes the confliction with nunjucks tags. It should fixed in #2461, will be released in next build.

**Comment by drumfreak** [comment]
asdf asdfasdfasdfasf
